content/en/04-tutorials/02-pxl-scripts/02-script-dev-environment.md

```diff
@@ -79,3 +81,26 @@ localStorage.clear('px-custom-bundle-path')
 <svg title='' src='script-dev-env/dev-env-3.png'/>
 
 3. Reload the [Live UI](https://work.withpixie.ai) webpage.
+
+## Contribute Your Script (Optional)
+
+Over time, we hope that our [script repository](https://github.com/pixie-labs/pixie/tree/main/src/pxl_scripts) grows into a community driven knowledge base of tools to observe, debug, secure and manage applications.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/So4ep2mMcSI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### Step 1: File an issue
+
+File an [issue](https://github.com/pixie-labs/pixie/issues/new/choose) with an explanation of what use-case you are looking to address help us make sure these community scripts are broadly applicable and useful.
```

Comment:
Sentence seems kind of long.
How about:
"File an issue with an explanation of what use-case you are looking to address. This will help us make sure these community scripts are broadly applicable and useful."